### ISSUE TITLE
Erro: Suspensão da Gestão de Conferência Eletrônica.

### ISSUE THREAD

**Issue by DaviSiqueira** [pedro-teixeira/correios#467]
Alguém passando por este problema?
Percebi agora a tarde.

![Captura de Tela 2020-03-30 às 16 09 12](https://user-images.githubusercontent.com/4529729/77951959-491a8f80-72a1-11ea-826d-a08ce38f79f8.png)

**Comment by rafaelpatro** [comment]
Sim, aqui aparece a mensagem de _Suspensão da Gestão de Conferência Eletrônica_, com o código de erro 011. Mas está retornando os valores com sucesso.
Vc pode configurar o módulo para que, em alguns códigos de erro, retorne positivo, emitindo apenas um aviso.
É aqui... https://github.com/pedro-teixeira/correios/blob/bbfa63a26b687b8ba4ccbd38189174c541a78ec2/app/code/community/PedroTeixeira/Correios/etc/config.xml#L126

**Comment by DaviSiqueira** [comment]
Obrigado @rafaelpatro vou tentar aqui

**Comment by DaviSiqueira** [comment]
@rafaelpatro não entendi como fazer retornar positivo como você mencionou.

Poderia me dar uma ajuda?
